nirc_ehr/resources/queries/dbo/q_cages.sql

```diff
@@ -1,7 +1,7 @@
 SELECT LOC1.LOCATION_ID      AS location,
        LOC1.LOCATION_ID      AS status, -- using for location as cage is not extensible
        LOC1.LOCATION_NAME    AS cage,
-       LOC2.LOCATION_ID      AS room
+       LOC2.LOCATION_ID || ' - ' || LOC2.LOCATION_NAME      AS room
```

Comment:
Any reason to add room name here? ehrModules/cage.js concatenates room and cage values in location column, we had issues where it crossed the length of 24 for location column. I increased the length to 100. Not sure if this might exceed 100 as well in future.

Reply:
This is added here to match the room row in q_rooms.sql (ultimately ehr_lookups.rooms).  The reason I added this concatenation to q_rooms.sql was just to get the name of the room into LK somewhere.  We're not pulling it in by ETL anywhere else.  Looks weird in the housing reports to have a full name for cage and room is just a rowid-like number and no name anywhere.  Don't see another column in ehr_lookups.rooms to add the name so just concatenated it here.  Still maintains the uniqueness with the location_id in front.  Let me know if you want to discuss a different approach.

Reply:
I didn't realize we had extensible columns, including "name" for ehr_lookups.rooms. Thanks for clearing that up @ankurjuneja .  Backed out this change and added a fk in nirc_ehrCustomizer to use the name as the display name.  Also updated animal history location search to use room name.